src/common/m_mpi_common.fpp

```diff
@@ -254,6 +254,15 @@ contains
                             MPI_COMM_WORLD, ierr)
         end if
 
+#else
+
+        icfl_max_glb = icfl_max_loc
+
+        if (any(Re_size > 0)) then
```

Comment:
do we have an `if(viscous)` logical flag?

Reply:
Not as far as I know.. though we should. I see 82 instances of `Re_size > 0` so even if we did, it isn't used much.

```console
MFC:[chemfc-up] $ grep 'Re_size > 0' -R src/ | wc -l
      82
```

Maybe create a GH issue for this.

Reply:
yes please create a GH issue for someone to add this :D

Reply:
https://github.com/MFlowCode/MFC/issues/677